packages/protocol/contracts/verifiers/RiscZeroVerifier.sol

```diff
@@ -40,7 +43,7 @@ contract RiscZeroVerifier is EssentialContract, IVerifier {
         initializer
     {
         __Essential_init(_owner, _rollupAddressManager);
-        receiptVerifier = IRiscZeroReceiptVerifier(_receiptVerifier);
+        verifier = IRiscZeroVerifier(_receiptVerifier);
```

Comment:
we shall use account manager to resolve addresses instead of using a local variable.